cppapi/server/multiattribute.cpp

```diff
@@ -1061,7 +1067,10 @@ void MultiAttribute::remove_attribute(string &attr_name,bool update_idx)
 			{
 				int idx = dev_att_list[loop]->get_attr_idx();
 				if (idx > old_idx)
+				{
 					dev_att_list[loop]->set_attr_idx(idx - 1);
+					(*dev_ite)->get_device_attr()->ext->attr_map[dev_att_list[loop]->get_name_lower()].att_index_in_vector--;
```

Comment:
It is better to avoid such long lines. It is a hell to debug it and maintain. Just split it declaring some local variables:

```cpp
string attr_name_lower = dev_att_list[loop]->get_name_lower();
DeviceAttributeExt* ext = (*dev_ite)->get_device_attr()->ext;
ext->attr_map[attr_name_lower].att_index_in_vector--;

```